src/es_archiver/lib/directory.ts

```diff
@@ -18,10 +18,9 @@
  */
 
 import { readdir } from 'fs';
-
 import { fromNode } from 'bluebird';
 
-export async function readDirectory(path) {
-  const allNames = await fromNode(cb => readdir(path, cb));
+export async function readDirectory(path: string) {
+  const allNames = await fromNode<string[]>(cb => readdir(path, cb));
```

Comment:
NIT: should probably use `promisify` now. But As I said, the PR is only doing straightforward TS conversion.